src/pages/NotFound/index.tsx

```diff
@@ -0,0 +1,10 @@
+import { FC } from 'react';
+import { Typography } from '@mui/material';
+
+const NotFound: FC = () => (
+  <Typography variant="h3" align="center">
+    Not Found
```

Comment:
could you amend this page a little bit by adding a link to the homepage, please?
it seems not so informative to me - 
<img width="819" alt="image" src="https://user-images.githubusercontent.com/11280780/187211941-1172ce8a-188b-4773-b113-89b0adff8197.png">

Reply:
![image](https://user-images.githubusercontent.com/110384829/189128715-5d106d79-0196-4f7c-93d4-a0c93cd36046.png)